BUILD.gn

```diff
@@ -28,7 +28,8 @@ import("${chip_root}/build/chip/tools.gni")
 if (current_toolchain != "${dir_pw_toolchain}/dummy:dummy") {
   declare_args() {
     chip_enable_python_modules =
-        (current_os == "mac" || current_os == "linux") && host_cpu == "x64"
+        (current_os == "mac" || current_os == "linux") &&
+        (host_cpu == "x64" || host_cpu == "arm64")
```

Comment:
should we filter host cpu at all in this case?

I imagine currently this would filter 32-bit host CPUs ... is there value in that? I am not even sure 32bit mac or linux are used and if they are, would they really behave differently/badly?

Reply:
IIRC it doesn't work on arm32.

Reply:
Maybe we should consider `host_cpu != arm32` then. 
Either way, we probably do not care much.